.babelrc

```diff
@@ -0,0 +1 @@
+{ "presets": ["@babel/preset-env", "@babel/preset-react"]}
```

Comment:
Please delete this file, it is now babel.config.js with different syntax and will fix the problem

Reply:
done